mapfishapp/src/main/java/org/georchestra/mapfishapp/ws/classif/SLDClassifier.java

```diff
@@ -347,8 +347,8 @@ private FeatureTypeStyle createFeatureTypeStyle(I_FilterFactory filterFact, I_Sy
      * @throws DocServiceException 
      */
     @SuppressWarnings("unchecked")
-    private WFSContentDataStore connectToWFS(final URL wfsUrl) throws DocServiceException {   
-    	WFSContentDataStore wfs = null;
+    private WFSDataStore connectToWFS(final URL wfsUrl) throws DocServiceException {   
+    	WFSDataStore wfs = null;
```

Comment:
May need side effects testing ?

Reply:
not sure I follow. From the old geotools version we were using to the 19.x series this patch upgrades to, the WFS-NG datastore graduated as the default one and `WFSContentDataStore` is no longer visible/does no longer exist. I've updated the code to use the public classes `WFSDataStoreFactory`/`WFSDataStore` which are the same ones they used to be but renamed, same parameters, etc.